Description: Various spelling errors in docs and comments
Author: Carl Suster <carl@contraflo.ws>

Index: plowshare4/docs/plowdown.1
===================================================================
--- plowshare4.orig/docs/plowdown.1
+++ plowshare4/docs/plowdown.1
@@ -296,7 +296,7 @@ module name
 newline
 .TP
 \fI%s\fR
-destination (local) file size (positive interger in bytes).
+destination (local) file size (positive integer in bytes).
 Important: Empty string is returned when \fB--skip\-final\fR switch is specified.
 .TP
 \fI%t\fR
Index: plowshare4/docs/plowup.1
===================================================================
--- plowshare4.orig/docs/plowup.1
+++ plowshare4/docs/plowup.1
@@ -245,7 +245,7 @@ module name
 newline
 .TP
 \fI%s\fR
-filesize (positive interger in bytes)
+filesize (positive integer in bytes)
 .TP
 \fI%t\fR
 tabulation character
Index: plowshare4/src/modules/bitshare.sh
===================================================================
--- plowshare4.orig/src/modules/bitshare.sh
+++ plowshare4/src/modules/bitshare.sh
@@ -94,7 +94,7 @@ bitshare_download() {
         echo $((WAIT))
         return $ERR_LINK_TEMP_UNAVAILABLE
 
-    elif match "Sorry, you cant download more then [[:digit:]]\+ files\? at time\." "$RESPONSE"; then
+    elif match "Sorry, you can't download more than [[:digit:]]\+ files\? at time\." "$RESPONSE"; then
         return $ERR_LINK_TEMP_UNAVAILABLE
     fi
 
Index: plowshare4/src/modules/sendspace.sh
===================================================================
--- plowshare4.orig/src/modules/sendspace.sh
+++ plowshare4/src/modules/sendspace.sh
@@ -174,7 +174,7 @@ sendspace_delete() {
 
     PAGE=$(curl "$URL") || return
 
-    if match 'You are about to delete the folowing file' "$PAGE"; then
+    if match 'You are about to delete the following file' "$PAGE"; then
         FORM_HTML=$(grep_form_by_order "$PAGE" 3)
         FORM_URL=$(echo "$FORM_HTML" | parse_form_action)
         FORM_SUBMIT=$(echo "$FORM_HTML" | parse_form_input_by_name_quiet 'delete')
