Description: Fixes several minor typos in shell script comments
Author: Carl Suster <carl@contraflo.ws>
Forwarded: https://code.google.com/p/plowshare/issues/detail?id=1265

Index: plowshare4/src/core.sh
===================================================================
--- plowshare4.orig/src/core.sh
+++ plowshare4/src/core.sh
@@ -2577,7 +2577,7 @@ set_exit_trap() {
     trap exit_handler EXIT
 }
 
-# Check existance of executable in $PATH
+# Check existence of executable in $PATH
 # Better than "which" (external) executable
 #
 # $1: Executable to check
Index: plowshare4/src/download.sh
===================================================================
--- plowshare4.orig/src/download.sh
+++ plowshare4/src/download.sh
@@ -884,7 +884,7 @@ declare -r UMASK=$(umask)
 test "$UMASK" && umask 0066
 
 # Remember last host because hosters may require waiting between
-# sucessive downloads.
+# successive downloads.
 PREVIOUS_HOST=none
 
 # Only used when CACHE policy is session (default).
Index: plowshare4/src/modules/filemonkey.sh
===================================================================
--- plowshare4.orig/src/modules/filemonkey.sh
+++ plowshare4/src/modules/filemonkey.sh
@@ -90,7 +90,7 @@ filemonkey_upload() {
                     "$BASE_URL/manage") || return
 
                 if [ -z "$PAGE" ]; then
-                    log_error 'An error has occured. Remote folder alread exists?'
+                    log_error 'An error has occurred. Remote folder already exists?'
                     return $ERR_FATAL
                 fi
 
Index: plowshare4/src/modules/freakshare.sh
===================================================================
--- plowshare4.orig/src/modules/freakshare.sh
+++ plowshare4/src/modules/freakshare.sh
@@ -132,7 +132,7 @@ freakshare_download() {
         captcha_ack $ID
         log_debug 'correct captcha'
 
-        # Sorry, you cant download more then 1 files at time.
+        # Sorry, you can't download more then 1 files at time.
         if match 'download more then.*files at time.' "$PAGE"; then
             log_error 'No parallel download allowed'
             echo 120
Index: plowshare4/src/modules/ge_tt.sh
===================================================================
--- plowshare4.orig/src/modules/ge_tt.sh
+++ plowshare4/src/modules/ge_tt.sh
@@ -190,7 +190,7 @@ ge_tt_upload() {
             return $ERR_FATAL
         fi
 
-        log_debug "Wait for server to recieve the file... [$((TRY++))]"
+        log_debug "Wait for server to receive the file... [$((TRY++))]"
         wait 1 || return
     done
 
Index: plowshare4/src/modules/hipfile.sh
===================================================================
--- plowshare4.orig/src/modules/hipfile.sh
+++ plowshare4/src/modules/hipfile.sh
@@ -356,7 +356,7 @@ hipfile_upload() {
     log_debug "State: '$STATE'"
 
     if [ "$STATE" = 'OK' ]; then
-        log_debug 'Upload successfull.'
+        log_debug 'Upload successful.'
     elif [ "$STATE" = 'unallowed extension' ]; then
         log_error 'File extension is forbidden.'
         return $ERR_FATAL
Index: plowshare4/src/modules/ryushare.sh
===================================================================
--- plowshare4.orig/src/modules/ryushare.sh
+++ plowshare4/src/modules/ryushare.sh
@@ -83,7 +83,7 @@ ryushare_download() {
     PAGE=$(curl -c "$COOKIE_FILE" -b "$COOKIE_FILE" -b 'lang=english' "$URL") || return
 
     # The file you were looking for could not be found, sorry for any inconvenience
-    # The file was removed by adminstrator
+    # The file was removed by administrator
     if match 'File Not Found\|file was removed' "$PAGE"; then
         return $ERR_LINK_DEAD
     fi
